### PR TITLE
CLI: improve help and add autocompletion

### LINKED ISSUE #74: CLI: The in-command help function doesn't show all possible options for mint-native-token like reading metadata from a json file (which is available)
Actually check for all commands.

### LINKED ISSUE #140: CLI: investigate dialoguer completion feature

### PR DESCRIPTION
**Pull request by thibault-martinez**
Fixes https://github.com/iotaledger/iota-sdk/issues/74
Fixes https://github.com/iotaledger/iota-sdk/issues/140

### CODE REVIEW

File: cli/src/account_completion.rs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        "help",
+    ],
+};
+
+impl<'a> Completion for AccountCompletion<'a> {
+    fn get(&self, input: &str) -> Option<String> {
+        let matches = self
+            .options
+            .iter()
+            .filter(|option| option.starts_with(input))
```
Comment (kwek20) [inline cli/src/account_completion.rs:53]: maybe we want to switch this to contains? just typing power, would be nice that all options show up

Comment (thibault-martinez) [reply cli/src/account_completion.rs:53]: Yeah but you have to return only one String, so what do you return when you provide power ?

Comment (kwek20) [reply cli/src/account_completion.rs:53]: ah i see, thats silly. never mind then

File: cli/src/command/account.rs [hunk 2]
```diff
@@ -41,110 +41,182 @@ pub enum AccountCommand {
...
+    },
+    /// Burn an NFT.
+    BurnNft {
+        /// NFT ID to be burnt, e.g. 0xecadf10e6545aa82da4df2dfd2a496b457c8850d2cab49b7464cb273d3dffb07.
+        nft_id: String,
+    },
     /// Claim outputs with storage deposit return, expiration or timelock unlock conditions.
-    Claim { output_id: Option<String> },
+    Claim {
+        /// Output IDs to be claimed.
```
Comment (Thoralf-M) [inline cli/src/command/account.rs]:
```suggestion
        /// Output ID to be claimed.
```

**Review by Thoralf-M — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Pull request merged**